### PR TITLE
ENH: Build against ITK 5.3-RC3

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: .github/workflows/build-test-package.yml [hunk 1]
```diff
@@ -134,9 +134,9 @@ jobs:
     strategy:
       max-parallel: 2
       matrix:
-        python-version: [36, 37, 38, 39]
+        python-version: [37, 38, 39, 310]
         include:
-          - itk-python-git-tag: "v5.2.1.post1"
+          - itk-python-git-tag: "v5.3rc03"
```
Comment (tbirdso) [inline .github/workflows/build-test-package.yml:139]: Should these be moved to `env` variables? See [https://github.com/InsightSoftwareConsortium/ITKSplitComponents/blob/master/.github/workflows/build-test-package.yml#L5](https://github.com/InsightSoftwareConsortium/ITKSplitComponents/blob/master/.github/workflows/build-test-package.yml#L5)

**Review by tbirdso — COMMENTED** [review]
Why rc3 and not rc4?

File: .github/workflows/build-test-package.yml (hunk 1 above)
Comment (thewtex) [reply .github/workflows/build-test-package.yml:139]: Yes -- these updates are based on ITKModuleTemplate `main`. Could you please port the ITKSplitComponents updates to ITKModuleTemplate?

Comment (tbirdso) [reply .github/workflows/build-test-package.yml:139]: Please see [https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/pull/126](https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/pull/126).

**Review by tbirdso — COMMENTED** [review]
May also need to fix the Windows grep path issue, please see [https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/pull/126/files#diff-8b27c545141e883b71167a60f61e5a67a8776c84033f7aaf516fd7cafd2b4e02L290](https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/pull/126/files#diff-8b27c545141e883b71167a60f61e5a67a8776c84033f7aaf516fd7cafd2b4e02L290)

**Pull request merged**